py7zr/py7zr.py

```diff
@@ -644,15 +644,16 @@ def test(self) -> bool:
         """Test archive using CRC digests."""
         return self._test_digests()
 
-    def extractall(self, path: Optional[Any] = None) -> None:
+    def extractall(self, path: Optional[Any] = None, return_dict: bool = False) -> Optional[Dict[str, IO[Any]]]:
```

Comment:
Shall we keep `extractall(self, path)-> None` as is and introduce `readall(self, path) -> Dict[str, IO[Any]]`?